src/WebHook/Event.php

```diff
@@ -9,10 +9,12 @@
  * @method static USER_CREATION()
  * @method static USER_MODIFICATION()
  * @method static USER_DELETION()
+ * @method static USER_UPDATE_SUBSCRIPTIONS()
  */
 class Event extends Enum
 {
     public const USER_CREATION = UserEvents::USER_CREATED;
     public const USER_MODIFICATION = UserEvents::USER_UPDATED;
     public const USER_DELETION = UserEvents::USER_DELETED;
+    public const USER_UPDATE_SUBSCRIPTIONS = UserEvents::USER_UPDATE_SUBSCRIPTIONS;
```

Comment:
When is this used?

Reply:
IMO declaring the constant in `UserEvents` is useless, it should be done here directly.
I can see that you follow the path taken before, but I think we should either use enums, or not.
Using both final classes and enum does not make any sense.